rpcs3/Emu/Cell/PPUModule.h

```diff
@@ -273,6 +272,9 @@ class ppu_module_manager final
 	static const ppu_static_module sys_libc;
 	static const ppu_static_module sys_lv2dbg;
 	static const ppu_static_module static_hle;
+
+private:
+	inline static std::unordered_map<std::string, ppu_static_module*> static_module_map;
```

Comment:
```suggestion
	inline static std::unordered_map<std::string, ppu_static_module*> s_module_map;
```
we prefer the prefix `s_` for static variables, its mentioned in our code style guidelines.

Reply:
Ofcourse github doesn't do inbuilt refactoring...
I'll fix the name shortly